apps/examples/stack_protection/stack_protection_main.c

```diff
@@ -78,29 +83,69 @@ int stack_prot_main(int argc, char *argv[])
 #endif
 {
 	char ch;
+	char thread_type;
+	int tc_fd, ret_chk;
 	int i = 0;
+	bool type_kthread;
```

Comment:
It is just my opinion,, How about renaming `type_kthread` to `is_kthread`? In many cases, is_xxx is used for checking the type.

Reply:
Done